backstage/packages/template-cluster/content/gitops/clusters/capz/aks-appset.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
```

Comment:
Since the templates are going to change over time and people would have to build this image themselves, let's please remove all these duplicate files and put them into a step in the "build your own image" to copy that directory to here.

Reply:
Good idea! Still exist in the provided image but not in the main repo now. I have added the detail to the readme under the build your own image, step 3.